src/input/styled-components.js

```diff
@@ -365,7 +365,7 @@ export const InputContainer = styled<SharedPropsT>(
 function getInputColors($disabled, $isFocused, $error, colors) {
   if ($disabled) {
     return {
-      color: colors.contentSecondary,
+      color: colors.inputTextDisabled,
```

Comment:
We should stick to using semantic color tokens and slowly deprecate component-specific tokens.

Reply:
Does that mean we don't need to support `inputTextDisabled` token?

Reply:
I believe this token is still used in some form components.